### PR TITLE
Add sha256 command

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @jjhelmus not sure if it is worth adding this here. Can you review/close/merge this?

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,8 @@ package:
 source:
   fn: simplejson-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
-  # md5 is OK but sha256 is desirable.
+  # md5 is OK but sha256 is desirable. On OS X: `shasum -a 256 file-name.tar.gz`,
+  # On Linux: `sha256sum file-name.tar.gz`.
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Would add that `openssl sha256 <file>` works great for me. It supports a variety of checksums. So it is really easy. Also, it comes with the `openssl` package. So, it doesn't require OS specific strategies.

**Comment by PythonCHB** [comment]
so is the idea here to simply put examples in the comments in the example?

that seems reasonable.

and `openssl sha256 <file>` works on Windows, too?

**Comment by jakirkham** [comment]
That is a good question. I looked into it briefly last night. I think the answer is it should be possible, but we are not currently installing `openssl.exe` on Windows. However, we are install it on Unix. This PR ( https://github.com/conda-forge/openssl-feedstock/pull/4 ) should install `openssl.exe` on Windows. Though I would like to get some feedback from Windows experts to make sure this is ok.

**Comment by jakirkham** [comment]
So, new builds of `openssl` from `conda-forge` provide this functionality on Windows. As a result, it is possible to do this on all platforms. We still don't use `openssl` as a dependency of `python` on Windows. That's a complex issue ( https://github.com/conda-forge/python-feedstock/issues/8 ), which we are interested in solving. Though it seems totally ok to install this side-by-side with an existing `python` package in an environment based on some [testing]( https://github.com/conda-forge/openssl-feedstock/pull/4#issuecomment-234793535 ).

**Comment by jjhelmus** [comment]
I think recommending the `openssl` solution is the best option.  It is cross platform and seems more reliable that the other possibilities.

**Comment by ChrisBarker-NOAA** [comment]
Well, a python script is just as cross-platform and reliable -- though then we need to put it somewhere and make it available. So as long as teh openssl thing works well on Windows, we're good to go.

**Comment by ChrisBarker-NOAA** [comment]
So where should this be documented? It could go here:

https://conda-forge.github.io/#update_recipe

but looks too low-level for that -- do we have a "how to build and maintian a recipe" doc somewhere?

**Comment by ChrisBarker-NOAA** [comment]
I added a note about this in #1160 

Maybe merge that and close this? Someone else's call :-)

**Comment by jakirkham** [comment]
I'm happy either way.

**Comment by jakirkham** [comment]
Thanks @ocefpaf.

**Pull request merged**